src/pages/components/combobox.mdx

```diff
@@ -0,0 +1,204 @@
+---
+title: Combobox
+description: Combobox is an input plus a list box that appears beneath it. It is used for search and filtering.
```

Comment:
<img width="582" alt="Screen Shot 2021-12-03 at 12 01 50 PM" src="https://user-images.githubusercontent.com/1975639/144658387-b0aa05a6-ebbe-4f7c-9f1f-aeea5f7d0a2a.png">
Could we use active voice?

Reply:
@diana-oum what are your thoughts here?

Reply:
@Francois-Esquire - "It searches and filters down options for the user."

I also think "Input" is actually the name of a component. Shouldn't that be linked?

Reply:
Thank you @diana-oum ! That works, will update.